geotrek/api/v2/urls.py

```diff
@@ -80,6 +80,7 @@
     router.register('signage_sealing', api_views.SealingViewSet, basename='signage-sealing')
     router.register('signage_color', api_views.ColorViewSet, basename='signage-color')
     router.register('signage_direction', api_views.DirectionViewSet, basename='signage-direction')
+    router.register('signage_conditions', api_views.SignageConditionViewSet, basename='signage-condition')
```

Comment:
```suggestion
    router.register('signage_condition', api_views.SignageConditionViewSet, basename='signage-condition')
```
I think we should use singular `signage_condition` in api v2 since all routes have singular labels